db/db-challenge3/db-001/db-001.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO users (id, email, password, name, introduction, office_phone_number, phone_number, is_deleted, created_at, updated_at) VALUES
+(1, 'admin@labot.inc', 'admin', 'admin', 'このスペースの管理者です。', '00-0000-0000', NULL, 0, '2020/04/20 15:47:52', '2020/04/20 15:47:52'),
+(2, 'mysql@labot.inc', 'mysql55', 'mysql', NULL, NULL, '080-1234-1253',	0, '2020/04/22 16:02:55', '2020/04/22 16:02:55'),
+(3, 'sqlserver@labot.inc', 'apple',	'sqlserver', 'オラクルデータベースが好きです。', NULL, NULL, 0, '2020/04/22 17:32:55', '2020/04/22 17:32:55'),
+(4, 'mariadb@labot.inc', 'maria', 'mariadb', NULL, '03-6666-5879', NULL, 0, '2020/04/22 18:45:21', '2020/04/22 18:45:21'),
+(5, 'laravel@labot.inc', 'lara', 'laravel', NULL, NULL, NULL, 0, '2020/04/22 18:46:01',	'2020/04/22 18:46:01'),
+(6, 'cakephp@labot.inc', 'cake89', 'cake', NULL, '03-987-1234', '090-5555-7854', 0,	'2020/04/23 02:51:06', '2020/04/25 11:45:06'),
+(7, 'react@labot.inc', 'node', 'react', NULL, NULL, NULL, 0, '2020/04/23 09:54:32',	'2020/04/23 09:54:32'),
+(8, 'html@labot.inc', 'css3', 'html5', NULL, NULL, '080-5849-5555',	0, '2020/04/23 09:55:14', '2020/04/23 09:55:14'),
+(9, 'php@labot.inc', 'php7', 'php',	'伝説のphp6', NULL, NULL, 0, '2020/04/24 01:01:06',	'2020/04/24 01:01:06'),
+(10, 'javascript@labot.inc', 'react', 'js',	'Hello world', '0120-555-458', NULL, 0,	'2020/04/24 05:03:58', '2020/04/24 05:03:58'),
+(11, 'microsoft@gmail.com',	'google', 'amazon',	'facebook', NULL, '080-9999-5555', 1, '2020/04/24 06:00:02', '2020/04/26 12:13:43')
```

Comment:
```suggestion
(11, 'microsoft@gmail.com',	'google', 'amazon',	'facebook', NULL, '080-9999-5555', 1, '2020/04/24 06:00:02', '2020/04/26 12:13:43');
```

FYI
今回は「;」なくても、phpMyAdminが自動的に文末と判断するので動作はします。
ただ、サーバー等では文の終わりをきちんと定義しないと、MySQL文が実行されません。

細かいですが、きちんとつける癖をつけておきましょう。